modules/cnf-understanding-workload-hints.adoc

```diff
@@ -62,3 +62,122 @@ perPodPowerManagement: true
     | Allows for power management per pod.
 
 |===
+
+This table describes the list of kernel arguments configured by applying the workload hints. The kernel arguments apply to all profiles with the exceptions called out in notes in the description field.
+
+[cols="33%,67",options="header"]
+|===
+| Argument | Description 
+
+| `skew_tick=1`
+a|Configures the kernel to adjust the timer skew. This parameter adjusts the timing of timer interrupts to synchronize with the CPU frequency and reduce timing errors.
+
+
+| `tsc=reliable`
+a| Specifies that the timestamp counter (TSC) should be used as a reliable time source. The TSC is a high-resolution per-CPU counter used for performance monitoring and timing purposes.
+
+
+| `rcupdate.rcu_normal_after_boot=1`
+a| This setting ensures that read-copy update (RCU) `normal` state is deferred until after the system boot process completes, potentially optimizing boot performance and avoiding interference with other boot-time tasks.
```

Comment:
```suggestion
a| This setting ensures that read-copy update (RCU) `normal` state is used until after the system boot process completes, optimizing boot performance and avoiding interference with other boot-time tasks.
```
The it's the grace-period primitives that are deferred, and instead `normal` state is used